cluster/src/assembly/resources/conf/iotdb-cluster.properties

```diff
@@ -66,8 +66,11 @@ max_unsnapshoted_log_size=134217728
 # whether to use batch append entries in log catch up
 use_batch_in_catch_up=true
 
-# max number of committed logs to be saved
-max_number_of_logs=100
+# minimum number of committed logs in memory
+min_num_of_logs_in_mem=100
+
+# maximum number of committed logs in memory
+max_num_of_logs_in_mem=1000
```

Comment:
Add more specific comments like
 "# minimum number of committed logs in memory, after each log deletion, at most such number of logs will remain in memory. Increasing the number will reduce the chance to use snapshot in catch-ups, but will also increase memory footprint"
"# maximum number of committed logs in memory, when reached, a log deletion will be triggered. Increasing the number will reduce the chance to use snapshot in catch-ups, but will also increase memory footprint"

Reply:
Sure, that will be more clear.